workflow.txt

```diff
@@ -1,10 +1,9 @@
+fork on github
 git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
+git checkout master
 git add .
+git commit -m ""
 git pull upstream master
```

Comment:
If you're going to pull, you should do it before you start committing. That way you're working the latest version of the code before you make your changes.